### DISCUSSION TITLE
Is there a way to pop-up the greeting message, only on certain pages?

### DISCUSSION THREAD

**Discussion by rozenmd** [papercups-io/papercups#725] [Ideas]
I saw y'all recently added the ability to pop-up the greeting message, is it possible to pick which pages it should do it on?

My use-case is to only pop-up the greeting message when the user navigates to the pricing page.

**Comment by reichert621** [comment]
hey @rozenmd! are you using our React component, or the html script snippet?

I think it should be possible with the React component, but it might not work with the html script in a SPA 🤔

**Reply by a8t** [reply]
Why not @reichert621 ? I wonder if there's some way to expose a global config object. Does the script already have that?

**Reply by rozenmd** [reply]
Using the react component - might require a bit more debugging on my end, but looks like it works only when the first page load is the page we want to pop up on. If they load say `/` then navigate to `/pricing`, it doesn't show. 

I'm using next.js, which might be complicating things.

**Reply by reichert621** [reply]
@rozenmd our website also uses next.js so i gave it a shot -- this seems to work: https://github.com/papercups-io/papercups.io/pull/23/files#diff-7d00022830ba75a86c65e4c942eb140aecdcf1492ba7ba3ed0f0dcee9e6a056eR253-R256

i'd like to figure out a way to make this a bit easier, but let me know if that works for you too!

**Reply by rozenmd** [reply]
Thanks @reichert621 ! 

Issue was in where I was rendering the component. I didn't realise that if you place it in `_app.tsx` it only renders once (on page load).

Moving the Papercups component into my Layout component fixed it, cheers! 🎉
